packages/base-map/src/index.tsx

```diff
@@ -112,68 +125,64 @@ const BaseMap = ({
   );
 
   return (
-    <MapProvider>
-      <Map
-        // eslint-disable-next-line react/jsx-props-no-spreading
-        {...mapLibreProps}
-        ref={passedRef}
-        id="mainMap"
-        latitude={viewState.latitude}
-        longitude={viewState.longitude}
-        mapLib={maplibregl}
-        mapStyle={baseLayer}
-        maxZoom={maxZoom}
-        onClick={onClick}
-        onContextMenu={onContextMenu}
-        onMove={evt => setViewState(evt.viewState)}
-        onTouchStart={() => {
-          setFakeMobileHover(false);
-        }}
-        style={{
-          display: "block",
-          height: forceMaxHeight ? "90vh" : "100%",
-          width: "100%"
-        }}
-        zoom={viewState.zoom}
-      >
-        {toggleableLayers.length > 0 && (
-          <Styled.LayerSelector
-            onTouchEnd={() => {
-              setFakeMobileHover(true);
-            }}
-            className="filter-group"
-            id="filter-group"
+    <Map
+      // eslint-disable-next-line react/jsx-props-no-spreading
+      {...mapLibreProps}
+      latitude={viewState.latitude}
+      longitude={viewState.longitude}
+      mapLib={maplibregl}
+      mapStyle={baseLayer}
+      maxZoom={maxZoom}
+      onClick={onClick}
+      onContextMenu={onContextMenu}
+      onMove={evt => setViewState(evt.viewState)}
+      onTouchStart={() => {
+        setFakeMobileHover(false);
+      }}
+      ref={passedRef}
+      style={{
+        display: "block",
+        height: forceMaxHeight ? "90vh" : "100%",
+        width: "100%"
+      }}
```

Comment:
Per my comment about the `forceMaxHeight` prop, pass in the `style` prop instead. Also pass the `className` prop so that styles can be overwritten by CSS or by the style prop. This component renders as a `<div>`, so by default it's `display: block`, which sets the `width` to 100% already.